### PR TITLE
added html file from trials for review

### PR DESCRIPTION
**Pull request by Itswali**
### My partner deleted the repository in which I gave the review for best practices for JavaScript that's why I provided link for this repository as we cannot submit empty link.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html style=" background-color:aqua;
+text-align:center;
+" >
```
Comment (Itswali) [inline index.html:4]: use to styles in separate css file for better and efficient use don't use inline css.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    <head>
+    
+    <link rel="stylesheet" href="project-1.css">
```
Comment (Itswali) [inline index.html:7]: the separate css file is connected but no line of codes is used in the file. use the file to apply styles in  web page for more appealing.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    <button id="submit" >submit</button>
+      
+      
+      </form>
+    
+      </div>
+    
+      </body>
+    
+      </html>
```
Comment (Itswali) [inline index.html:66]: Format the document in a proper manner so Anyone can easily read and understand it.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    </input>
+    
+    <br><br>
+    <textarea rows="4" cols=""> Enter Commments</textarea>
+    </input>
+    <br><br>
+    <button id="submit" >submit</button>
+      
+      
+      </form>
```
Comment (Itswali) [inline index.html:60]: use the section form to apply style and for clean and more documented code.

Comment (Itswali) [inline index.html:60]: Use the linters to check the errors in your code you didn't use any linters.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+<html style=" background-color:aqua;
+text-align:center;
+" >
+    <head>
+    
+    <link rel="stylesheet" href="project-1.css">
+    
+    </head>
+    <body>
+    <div class="mid" >
```
Comment (Itswali) [inline index.html:11]: Use the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs. For proper understanding.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+            <option value="2">NOT-A-Programmer</option>
+    
+        </select>
+        <br>
+    <input name="radio" value="1" type="radio" >Male</input>
+    
+    <input name="radio" value="2" type="radio" >Female</input>
+    
+    
+    <br><br>
```
Comment (Itswali) [inline index.html:44]: use the css property to add spaces in document.
The CSS is a better choice to style your web.

**Review by Itswali — COMMENTED** [review]
I just reviewed my projects.

**Review by frank1738 — APPROVED** [review]
Hi @Itswali , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


YesYouAreGIF (2)


## Highlights

✅ Correct use of Github inline comments
✅ Provided a reason as to why a change is required

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**